### PR TITLE
Create WS-C6513-E.yaml

### PR DESCRIPTION
**Pull request by bbrk364**
updating the module-bays

### CODE REVIEW

File: device-types/Cisco/WS-C6513-E.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    maximum_draw: 2800
+  - name: PS2-1
+    type: iec-60320-c20
+    maximum_draw: 2800
+  - name: PS2-2
+    type: iec-60320-c20
+    maximum_draw: 2800
+  - name: PS2-3
+    type: iec-60320-c20
+    maximum_draw: 2800
```
Comment (DanSheps) [inline device-types/Cisco/WS-C6513-E.yaml:34]: This should be made modular with appropriate PSU ports depending on the PSU inserted

File: device-types/Cisco/WS-C6513-E.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+u_height: 19
+is_full_depth: true
+subdevice_role: parent
+console-ports:
+  - name: con0
+    type: rj-45
+interfaces:
+  - name: FastEthernet1
+    type: 100base-tx
+    mgmt_only: true
```
Comment (DanSheps) [inline device-types/Cisco/WS-C6513-E.yaml:15]: This should be part of the SUP card, should be modular and slotted in with the rest of the modules

File: device-types/Cisco/WS-C6513-E.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+manufacturer: Cisco
+model: Catalyst 6513
+slug: ws-c6513
+part_number: WS-C6513
+u_height: 19
+is_full_depth: true
+subdevice_role: parent
+console-ports:
+  - name: con0
+    type: rj-45
```
Comment (DanSheps) [inline device-types/Cisco/WS-C6513-E.yaml:11]: This should be part of the SUP card, should be modular and slotted in with the rest of the modules

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Do you still need this? @bbrk364

**Pull request closed without merging**